src/cljs/main/broadfcui/page/profile.cljs

```diff
@@ -38,9 +38,9 @@
 (defn get-url-search-param [param-name]
   (.get (js/URLSearchParams. js/window.location.search) param-name))
 
-(defn auth-url-link [href text]
+(defn auth-url-link [href text provider]
   (if href
-    (links/create-external {:href href :target "_self"} text)
+    (links/create-external {:href href :target "_self" :data-test-id provider :id "provider-link"} text)
```

Comment:
IIRC, the `data-test-id` attributes only get rendered on testing environments?  Not sure if that's something you need to add logic for here or if the UI code knows when to ignore these attributes based on whatever environment it's running on.  Maybe @asingh7115 knows the answer?  Or maybe ask a UI person.

Reply:
Just asked her, it shows up on all environments, is that okay?